enginetest/queries/query_plans.go

```diff
@@ -1906,8 +1906,8 @@ var PlanTests = []QueryPlanTest{
 	{
 		Query: `SELECT ROW_NUMBER() OVER (ORDER BY s2 ASC) idx, i2, s2 FROM othertable WHERE s2 <> 'second' ORDER BY i2 ASC`,
 		ExpectedPlan: "Sort(othertable.i2 ASC)\n" +
-			" └─ Project(row_number() over ( order by [othertable.s2, idx=0, type=TEXT, nullable=false] ASC) as idx, othertable.i2, othertable.s2)\n" +
-			"     └─ Window(row_number() over ( order by [othertable.s2, idx=0, type=TEXT, nullable=false] ASC), othertable.i2, othertable.s2)\n" +
+			" └─ Project(row_number() over ( order by [othertable.s2, idx=0, type=VARCHAR(20), nullable=false] ASC) as idx, othertable.i2, othertable.s2)\n" +
```

Comment:
Now that I'm looking at this, this looks wrong -- the order by should not have that level of detail

Something is calling DebugString instead of String